pkg/execution/executor/executor.go

```diff
@@ -1528,55 +1528,63 @@ func (e *executor) HandleInvokeFinish(ctx context.Context, evt event.TrackedEven
 }
 
 // Cancel cancels an in-progress function.
-func (e *executor) Cancel(ctx context.Context, runID ulid.ULID, r execution.CancelRequest) error {
-	s, err := e.sm.Load(ctx, runID)
+func (e *executor) Cancel(ctx context.Context, id sv2.ID, r execution.CancelRequest) error {
+	md, err := e.smv2.LoadMetadata(ctx, id)
+	if err == sv2.ErrMetadataNotFound {
+		logger.From(ctx).Warn().Err(err).Msg("cancelling function already removed")
```

Comment:
do we need this log? it's not logging any tenant identifiers so it's kinda useless if it's gonna be used for identifying runs.
if no additional metadata is gonna be added, let's just remove it.